### ISSUE TITLE
invités

### ISSUE THREAD

**Issue by HerKa2** [outilssgdf/analytiscout#68]
S,
Pourrais-tu ajouter les "invité"s dans l'analyse des jeunes ?
Merci.

**Comment by HerKa2** [comment]
jeunes et responsables ?

**Comment by sbouchex** [comment]
non, un invité n'a pas de code adhérent donc compliqué à intégrer.
